doc/user/hierarchy.xml

```diff
@@ -602,7 +631,7 @@ SConscript('src/SConscript', exports='env')
       <para>
 
       These calls export the specified variables
-      to only the listed &SConscript; files.
+      to only the listed &SConscript; file.
```

Comment:
Can be files... (plural)

Reply:
I think the very next sentence makes that clear.

Reply:
maybe ```file(s)``` ?

Reply:
ok.